flow/scripts/sc_apr.tcl

```diff
@@ -0,0 +1,117 @@
+source sc_manifest.tcl
```

Comment:
More comments: purpose of file, description of each section and individual steps